install_config/syncing_groups_with_ldap.adoc

```diff
@@ -229,7 +229,7 @@ $ oadm groups prune --sync-config=config.yaml --confirm
 ----
 
 [[sync-examples]]
-== Sync Examples
+== Direct Membership Sync Examples
```

Comment:
A minority of people need nested groups. Don't make everyone parse this out. Let the people who need nested groups read your note and click your link.